calcom/cal.com
# Hotfix: blank page for booking embed in Incognito Chrome

## What does this PR do?


Fixes #2618


## Type of change




## How should this be tested?



- [ ] Open Incognito Chrome. Make sure that "Block third party cookies in Incognito is checked". This is default for Chrome Incognito
<img width="756" alt="Screenshot 2022-05-09 at 3 14 32 PM" src="https://user-images.githubusercontent.com/1780212/167384517-c580c894-c896-4447-8aa8-0234a4dee947.png">. Observe that if you have an embed on a page it doesn't load(ensure that embed and your website domains are different). After this fix, it would workk
